Preloader.class.php

```diff
@@ -46,22 +46,11 @@ static function sourceText( $page ) {
 				->getRevisionByTitle( $title );
 
 			$content = $revisionRecord->getContent( SlotRecord::MAIN );
-			$text = ContentHandler::getContentText( $content );
-
-			return self::transform( $text );
+			parserOptions = ParserOptions::newFromUser( $wgUser );
```

Comment:
This global is not loaded, additionally it is deprecated. Should use `$user = RequestContext::getMain()->getUser();` above and `$user` here. Also there is a syntax error here, the parserOptions variable is missing a `$`.